File: frontend/src/main/web/src/app/employees/employee.ts
import { Booking } from '../bookings/booking';

export class Employee {
  employeeId: string;
  name: string;
  surname: string;
  role: string;
  depto: string;
  bookings: Booking[];

 fullName(): string {
   return name + ' ' + this.surname;
 }
}

File: backend/src/main/java/fonobook/models/Employee.java
package fonobook.models;

import lombok.*;

import javax.persistence.*;
import java.util.List;

@Entity
@Data
@ToString
@EqualsAndHashCode
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class Employee {
    @Id
    @GeneratedValue
    private Long id;
    private String employeeId;
    private String name;
    private String surname;
    private String role;
    private String depto;
    @OneToMany
    private List<Booking> bookings;
}
